app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def logged_in?
+    !!session[:user_id]
+  end
```

Comment:
This probably belongs in your ApplicationController base class rather than a helper. Helpers are designed to help you out with building your views.